napari/qt/__init__.py

```diff
@@ -1,13 +1,34 @@
+import warnings
+
 from .._qt.qt_event_loop import get_app, run
 from .._qt.qt_main_window import Window
 from .._qt.qt_resources import compile_qt_svgs, get_stylesheet
 from .._qt.qt_viewer import QtViewer
-from .._qt.widgets.qt_viewer_buttons import QtViewerButtons
+from .._qt.widgets.qt_viewer_buttons import QtStateButton, QtViewerButtons
 from .threading import create_worker, thread_worker
 
+
+class QtNDisplayButton(QtStateButton):
+    def __init__(self, viewer):
+        warnings.warn(
+            "QtNDisplayButton is deprecated and will be removed in 0.4.9",
```

Comment:
Hi @Czaki , thanks for working on this. Could you make the string localizable?

```
from ..utils.translator import trans

trans._("QtNDisplayButton is deprecated and will be removed in 0.4.9")
```

Thanks!

Reply:
Are you sure that warnings should be translated? I think that this should be only applied to the interface elements.

Reply:
It's probably helpful for people that they get translated, no? We might not have done it everywhere yet, but it seems reasonable to me. Thoughts @goanpeca?